integrations/whatsapp/src/message-types/card.ts

```diff
@@ -119,6 +117,9 @@ function createButtons(nonURLActions: NonUrlAction[]) {
       case 'say':
         buttons.push(button.create({ id: `${SAY_PREFIX}${action.value}`, title: action.label }))
         break
+      case 'url':
+        // TODO implement support for URL actions
+        throw new Error('URL actions not implemented yet')
```

Comment:
**nitpick:** if you throw a `RuntimeError` from `@botpress/client` here, it will be displayed correctly to the bot